Makefile

```diff
@@ -50,7 +50,7 @@ bash-airflow:
 	docker-compose run --rm ${service_name}-airflow /bin/bash
 
 build-docs:
```

Comment:
Are we using the `Makefile` anymore?

Reply:
I think it would be nice to have for local devs. I don't believe build-docs is part of a build.